terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/WebDriverEventListenerAdapter.java

```diff
@@ -83,4 +83,12 @@ public void beforeScript(String arg0, WebDriver arg1) {
     public void onException(Throwable arg0, WebDriver arg1) {
     }
 
+    @Override
+    public void afterNavigateRefresh(WebDriver arg0) {
+    }
+
+    @Override
+    public void beforeNavigateRefresh(WebDriver arg0) {
+    }
+

```

Comment:
@bthondasui 
- [ ] Please check dependency `selenium-support` version. This change is unnecessary.

It seems to be mixed version 2.46.0 and 2.53.1 in selenium series. Please see #526 .